contracts/test/contract.OperatorRegistry.test.js

```diff
@@ -0,0 +1,140 @@
+const OperatorRegistry = artifacts.require("OperatorRegistry");
+const DEFAULT_ADDRESS_INDEX = 0;
+const NON_OWNER_ADDRESS_INDEX = 1;
+
+contract("OperatorRegistry", (accounts) => {
+  const verifiedShareOperatorEOAs = [
+    accounts[1],
+    accounts[2],
+    accounts[3],
+    accounts[4],
+  ];
+  specify("Contract initialization", async () => {
+    const operatorRegistry = await OperatorRegistry.new();
+
+    await operatorRegistry.initialize(
+      verifiedShareOperatorEOAs /* shareEOAOperators_ */
+    );
+    assert.equal(
+      accounts[DEFAULT_ADDRESS_INDEX],
+      await operatorRegistry.owner()
+    );
+  });
+
+  specify(
+    "Add new verified operator address to the registry",
+    async () => {
+      const operatorRegistry = await OperatorRegistry.deployed();
+      const newShareVerifiedOperator = accounts[5];
+
+      await operatorRegistry.initialize(
+        verifiedShareOperatorEOAs /* shareEOAOperators_ */
+      );
+      operatorRegistry.addVerifiedOperator(newShareVerifiedOperator);
+
+      assert(
+        operatorRegistry.isOperator(newShareVerifiedOperator),
+        "New SHARE operator address not properly added to registry."
+      );
+    }
+  );
+
+  specify(
+    "Remove a verified operator address from the registry",
+    async () => {
+      const operatorRegistry = await OperatorRegistry.deployed();
+      const retiredShareVerifiedOperator = accounts[5];
+      await operatorRegistry.initialize(
+        verifiedShareOperatorEOAs /* shareEOAOperators_ */
+      );
+
+      assert(
+        operatorRegistry.isOperator(retiredShareVerifiedOperator),
+        `Operator address ${retiredShareVerifiedOperator} does not exist in registry`
+      );
+
+      const retiredOperation =
```

Comment:
Why `retired` here?

Reply:
It represents an address being decommissioned, do you suggest a simpler name?

Reply:
@DeusExM4ch1n4 I'd consider `addressToRemove` and `success = operator.removeVerifiedOperator(addressToRemove); assert(success, ...)`